purchases/src/main/kotlin/com/revenuecat/purchases/PurchasesOrchestrator.kt

```diff
@@ -556,18 +556,20 @@ internal class PurchasesOrchestrator(
     fun getCustomerInfo(
         callback: ReceiveCustomerInfoCallback,
     ) {
-        getCustomerInfo(CacheFetchPolicy.default(), callback)
+        getCustomerInfo(CacheFetchPolicy.default(), false, callback)
```

Comment:
For readability, might be good to specify the parameter name here
```suggestion
        getCustomerInfo(CacheFetchPolicy.default(), trackDiagnostics = false, callback)
```